CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ This changelog was created using the `clu` binary
 
 ### Improvements
 
+- (rpc) [#22](https://github.com/evmos/os/pull/22) Add server implementation.
```

Comment:
nit: I believe this is more appropriate since it is also a standalone package
```suggestion
- (server) [#22](https://github.com/evmos/os/pull/22) Add server implementation.
```